typescript.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+module.exports = {
+	extends: [
+		'plugin:@typescript-eslint/recommended'
+	],
+	plugins: [
+		'@typescript-eslint'
+	],
+	parserOptions: {
+		parser: '@typescript-eslint/parser',
```

Comment:
"parser" should be at the root level, not inside "parserOptions"

Reply:
Mh, I'm actually not sure about this. IIRC, then this is used in conjecture with vue files that are compiled before being committed and that have a TypeScript `<script>` tag. And then the root-level parser uses `vue-eslint-parser` and the one in parserOptions uses `@typescript-eslint/parser`.

But it has been ages that I worked with such a setup. Let's put it into the root and figure this case out when it arises.

Also, https://typescript-eslint.io/packages/parser/ does not specify any of the existing parser options, so I dropped them all as well.

Reply:
... or at least that is what I thought. It seems that at least 
```
		ecmaFeatures: {
			impliedStrict: true
		}
```
has an effect? I've added those options back. Clearly, I don't fully understand how they work and where they are documented yet.